frontend/src/constants/classifications.ts

```diff
@@ -0,0 +1,6 @@
+export enum CLASSIFICATIONS {
+  CoreOperational,
```

Comment:
You'll need to set the correct int value for each.

Reply:
@Fosol it sets them by default to 0, 1, 2, 3 unless otherwise specified